nikola/plugins/task/indexes.py

```diff
@@ -92,7 +92,8 @@ def get_path(self, classification, lang, dest_type='page'):
         """Return a path for the given classification."""
         if dest_type == 'rss':
             return [self.site.config['RSS_PATH'](lang)], True
-        # 'page' (index) or 'feed' (Atom)
+        if dest_type == 'feed':
+            return [self.site.config['ATOM_PATH'](lang)], True
```

Comment:
Please also change the `True` to `"auto"` here.

Reply:
Is `'auto'` ok or exactly `"auto"`? I think it makes no difference and is better in this case.

Reply:
Ah sorry, I wanted to write `'auto'` here as well :)